src/PKSim.Core/Model/CompoundPK.cs

```diff
@@ -1,37 +1,89 @@
-﻿using OSPSuite.Utility.Collections;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Collections;
 
 namespace PKSim.Core.Model
 {
    /// <summary>
-   /// Reperesents a cache containing specific PK-Values calculated for a given compound
+   /// Represents a cache containing specific PK-Values calculated for a given compound
    /// </summary>
    public class CompoundPK
    {
+      private readonly Cache<string, QuantityPKParameter> _quantityPKParameters = new Cache<string, QuantityPKParameter>(getKey:x => x.Name);
+
+      public CompoundPK()
+      {
+         bioAvailabilities = new Cache<int, double?>();
+         ddiAucInf = new Cache<int, double?>();
+         ddiCMax = new Cache<int, double?>();
+      }
       public string CompoundName { get; set; }
-      
+
       /// <summary>
-      /// AUC_inf of plasma curve following a IV infustion of 15 min
+      /// AUC_inf of plasma curve following a IV infusion of 15 min
       /// </summary>
-      public double? AucIV { get; set; }
+      private Cache<int, double?> bioAvailabilities { get; set; }
```

Comment:
private=>naming is off. Create a _privateMember (not get set)

Reply:
I think I don't need these separate private ones at all